client/package.json

```diff
@@ -24,7 +24,7 @@
     "build:workers": "cross-env NODE_OPTIONS=\"--max-old-space-size=7168\" webpack --config ./webpack-workers.js",
     "clean": "gatsby clean",
     "predevelop": "tsc -p ../tools/ && node ../tools/scripts/build/ensure-env.js && npm run build:workers -- --env development",
-    "develop": "cross-env NODE_OPTIONS=\"--max-old-space-size=4000\" gatsby develop --inspect=9230",
+    "develop": "cross-env NODE_OPTIONS=\"--max-old-space-size=5000\" gatsby develop --inspect=9230",
```

Comment:
What is this change?

Reply:
Gatsby kept running out of memory locally, so I pushed it up a bit.